RestoreTestKBFromProd/Program.cs

```diff
@@ -157,8 +194,20 @@ private async Task<Operation> MonitorOperation(Operation operation)
 
             return operation;
         }
-
         // </MonitorOperation>
+
+        private async Task RestoreTestKbForMultipleLanguages(string kbId)
+        {
+            var qna = new QnADTO() { Answer = "dummy answer", Questions = new List<string> { "dummy question" } };
```

Comment:
Nit: Can add a comment - we need to add a dummy QnA to create the relevant test index for the KB. In subsequent steps we'll override this with actual Prod KB content.